plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -295,12 +294,12 @@ public List<AddFileEntry> getActiveFiles(TableSnapshot tableSnapshot, ConnectorS
         }
     }
 
-    private List<AddFileEntry> loadActiveFiles(TableSnapshot tableSnapshot, ConnectorSession session)
+    private List<AddFileEntry> loadActiveFiles(TableSnapshot tableSnapshot, MetadataEntry metadataEntry, ProtocolEntry protocolEntry, ConnectorSession session)
     {
-        try (Stream<AddFileEntry> entries = getEntries(
+        try (Stream<AddFileEntry> entries = getAddFileEntries(
```

Comment:
inline `getAddFileEntries`

Reply:
I don't quite follow this suggestion

Reply:
`getAddFileEntries` is a new method and here is the only call site.
inline it, because having it inlined does not make the code less readable